Photo-Fetcher/src/components/UI/ImageContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const ImageContainer = ({ src, alt, isGrayscale }) => {
+    return (
+        <div className="image-container">
+            <img
+                src={src}
+                alt={alt}
+                style={{ filter: isGrayscale ? "grayscale(100%)" : "none" }}
+            />
+            <div className="overlay">
+                <h2>Lukas Budimaier</h2>
```

Comment:
h1, h2, h3, h4, h5, h6, h7, h8, h9 -> por jo, duhet separation of cencerns/concepts -> parimi baze i solid principles